src/components/App/App.js

```diff
@@ -101,15 +142,16 @@ function App() {
 
     // Simulate fetching news data
     setTimeout(() => {
-      getItems()
+      getNews(input)
         .then((data) => {
-          // Assuming data is an array of news cards
-          setNewsCards(data.flat()); // Flatten the array of arrays
+          setNewsCards(data.articles);
+        })
+        .then(() => {
           setIsSearchLoading(false);
         })
+
         .catch((error) => {
-          console.error("Error fetching news:", error);
-          setIsSearchLoading(false);
```

Comment:
`setIsSearchLoading(false)` should be in block `finally` to stop loading effect in any case because if there is an error the loading will not be stopped